vault/resource_quota_rate_limit.go

```diff
@@ -153,6 +185,10 @@ func quotaRateLimitRead(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
+	if provider.IsAPISupported(meta, provider.VaultVersion120) {
```

Comment:
following the existing pattern but it's unclear what's the point of this: If the server supports the new fields it's going to return them, if it doesn't it won't, there's no point in checking the version

Reply:
also, should we verify if it's an ent Vault?